src/app/components/Spinner.scss

```diff
@@ -1,16 +1,25 @@
-@import '../../scss/fonts';
-@import '../../scss/mixins';
-@import '../../scss/variables';
+@import "../../scss/fonts";
+@import "../../scss/mixins";
+@import "../../scss/variables";
 
 .spinner {
   position: relative;
   top: 20px;
   left: 20px;
   width: 40px;
   height: 40px;
+  background-color: $orange-6;
   border-radius: 100%;
-  background-color: #c3d0e4;
   animation: pulse 1s ease-in-out infinite;
+
+  &.spinner--secondary {
+    background-color: $gray-3;
+  }
+
+  &.spinner-landing {
```

Comment:
instead of making a single use class here for the landing, wouldn't it be cleaner just wrapping the landing spinner in a wrapper div that positions the spinner on the right corner?

Reply:
In this case I decided to make a specific class as Spinner component already had position attributes. However the solution is better so I've just removed Spinner positions from css and place all Spinners inside positioned parent divs